src/renderer/lib/cast.js

```diff
@@ -130,22 +132,51 @@ function chromecastPlayer () {
     })
   }
 
+  function serveSubtitles (cb) {
```

Comment:
In all functions in this file callback arguments are called `callback`, probably best to keep this consistent.